snakeplane/plugin_factory.py

```diff
@@ -723,6 +723,9 @@ def chunk_ii_close(current_interface: object):
                         return
                     self._build_metadata(plugin)
 
+                if not plugin.initialized:
```

Comment:
Should we get rid of the 
```python
                    if not plugin.initialized:
                        return
```
that is on line 722? If we still need it (to prevent an error on line 724), maybe we should think of a more succinct check.

Reply:
Yeah we need both checks. If you have any suggests on a clean way to do that I'm all ears!